lemarche/stats/models.py

```diff
@@ -1,6 +1,36 @@
+from datetime import datetime, timedelta
+
 from django.db import models
 
 
+class TrackerQuerySet(models.QuerySet):
+    def env_prod(self):
+        return self.filter(env="prod", isadmin=False)
+
+    def siae_views_last_3_months(self, siae_slug):
+        return self.env_prod().filter(
+            action="load",
+            page=f"/prestataires/{siae_slug}/",
+            date_created__gte=datetime.now() - timedelta(days=90),
+        )
+
+    def siae_buyer_views_last_3_months(self, siae_slug):
+        return self.env_prod().filter(
+            action="load",
+            page=f"/prestataires/{siae_slug}/",
+            data__meta__user_type="BUYER",
+            date_created__gte=datetime.now() - timedelta(days=90),
+        )
+
+    def siae_partner_views_last_3_months(self, siae_slug):
+        return self.env_prod().filter(
+            action="load",
+            page=f"/prestataires/{siae_slug}/",
+            data__meta__user_type="PARTNER",
+            date_created__gte=datetime.now() - timedelta(days=90),
+        )
+
+
```

Comment:
T'es pas obligé mais tu peux même faire ça si tu veux :) 
```suggestion
    def siae_views_last_3_months(self, siae_slug, user_type):
        return self.env_prod().siae_views_last_3_months(siae_slug).filter(
            data__meta__user_type=user_type,
        )


```

Reply:
of course, je corrige :)